action.yml

```diff
@@ -32,6 +32,10 @@ inputs:
     description: >
       If set, fails build if registry-mirrors cannot pull image. If registry-mirrors is empty, this flag is ignored.
       Type: Boolean
+  verbosity:
+    default: debug
```

Comment:
Wouldn't it make sense to let it default to `verbosity=info`?

Reply:
Yeah - I tend to agree that `info` is a better default. But I didn't want to change the behaviour as I'm an outsider :)

Reply:
I think it's ok to change it 😊

Reply:
https://github.com/cloudbees-io/kaniko/pull/11/commits/3dd2297ad6804aefbd7224fc4bdb668c3542a839 changes it to `info` as default.